1_fetch.R

```diff
@@ -186,6 +186,27 @@ p1_targets_list <- list(
   tar_target(
     p1_ntw_adj_matrix,
     read_csv(p1_ntw_adj_matrix_csv,show_col_types = FALSE)
+  ),
+
+  # Download and unzip metabolism estimates
+  tar_target(
+    p1_metab_tsv,
+    {
+    metab_file <- download_sb_file(sb_id = "59eb9c0ae4b0026a55ffe389",
+                                   file_name = "daily_predictions.zip",
+                                   out_dir="1_fetch/out")
+    unzip(zipfile=metab_file, exdir = dirname(metab_file), overwrite=TRUE)
+    file.path(dirname(metab_file), "daily_predictions.tsv")
+    },
+    format="file" 
+  ),
+  
+  # Download and unzip metabolism estimates
+  tar_target(
+    p1_metab,
+    {
+      read_tsv(p1_metab_tsv, show_col_types = FALSE)
```

Comment:
Adding the renaming step here would eliminate the need to repeat this step twice (in `p2a_trn_do_zarr` and `p2a_trn_val_do_zarr`). If we use the metabolism estimates elsewhere in the pipeline down the road, it'd also be helpful to have the site names match our format. 

```suggestion
      read_tsv(p1_metab_tsv, show_col_types = FALSE) %>%
      # create a new column "site_id". This column is the same as site_name from the
      # original data, but the 'nwis_' before the site number is removed to match site naming
      # conventions used in our pipeline.
      mutate(site_id = str_replace(site_name, "nwis_", ""))
```

Reply:
Great idea.